.editorconfig

```
@@ -8,3 +8,4 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 end_of_line = lf
+quote_type = single
```

to format single qutes only for js files 😉 
```suggestion
end_of_line = lf
[*.js]
quote_type = single
```

---

Thank you so much!!! Works magic!